iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractTerminal.java

```diff
@@ -30,6 +33,8 @@ abstract class AbstractTerminal implements TerminalExt {
 
     protected int num = -1;
 
+    protected final List<RegulatingPoint> regulated = new ArrayList<>();
```

Comment:
Just wondering, when a regulated terminal is removed the list is not updated, so you might have too many regulating points?